framework/doc/content/source/auxkernels/FunctionArrayAux.md

```diff
@@ -0,0 +1 @@
+!template load file=stubs/moose_object.md.template name=FunctionArrayAux syntax=/AuxKernels/FunctionArrayAux
```

Comment:
Let's try not to add stubs. If this this is going to be a framework level object, let's add some documentation. It looks sloppy coming to an "under construction" page.